stats.js

```diff
@@ -0,0 +1,207 @@
+const apiKey = 'zQOIBdFR8M0sYuT0njD9ku8oL'; // Socrata API Key.
```

Comment:
This should probably come from config, preferably a secret

Reply:
Done-ish.. but I still need to go through some Terraform setup to establish the secret.